### ISSUE TITLE
Content Listing Filters on Tablet/Mobile

### ISSUE THREAD

**Issue by DrewAllen88** [DrewAllen88/Enhancements#77]
The word "Seach" is off.

![image](https://cloud.githubusercontent.com/assets/8258720/9259656/ac374704-41d0-11e5-9845-bddec48734dc.png)

**Comment by timproDev** [comment]
@DrewAllen88 Though the text is broken, the ask is to render this text so that it can accommodate variation in word length. The original design uses the words "sort and filter" and the component is styled and built to accommodate only that text. When then text is shortened, the text breaks. The height of the box that contains the text is determined by the amount of text fields in the filter component. This means that we may still run into a readability issue if the filter component is short in height and the text requires a higher character limit. I wonder if Design can come up with something here. Perhaps, on mobile, we simply use some sort of icon that means more options, or pull out menu.

For example, in place of Search, use:

![image](https://cloud.githubusercontent.com/assets/8249300/9640500/f1161fb2-517f-11e5-9fe5-c1860cd8e8a7.png)

**Comment by DrewAllen88** [comment]
@joyjamir what are your thoughts on this? We can discuss it.

**Comment by joyjamir** [comment]
An icon could work.  Not sure I love this one, but if we can find other examples, might be a good option.

**Comment by timproDev** [comment]
@DrewAllen88 @joyjamir is this a design task?

**Comment by joyjamir** [comment]
Yes, I think so.  But can we fix the current view in the interim?

**Comment by DrewAllen88** [comment]
Yeah we just have to change the text from "Search" to "Sort and Filter" or something just as long.

**Comment by DrewAllen88** [comment]
Minimum height (three cards)
![search_sort_filter gif](https://cloud.githubusercontent.com/assets/8258720/9771749/14f8a126-5706-11e5-96b7-2dd49efaacb3.png)

@timproDev

**Comment by DrewAllen88** [comment]
Perhaps something to go below text and above tiles
![image](https://cloud.githubusercontent.com/assets/8258720/10051699/dc6c0fbe-61f0-11e5-9941-c20a869534f9.png)

**Comment by DrewAllen88** [comment]
Set a minimum height of roughly 1.3 card lengths; Character max limit is 25, let's go with a minimum of 5; min and max chars need to be tested; the sort box gets taller with additional search parameters; all test should be in caps.

**Comment by timproDev** [comment]
This has been updated. The character limit accommodates for max 25 characters. The filter box height has been increased to a min-height; it should expand should there be more filters included. this will be included in the Release 3 FED deliverable

**Comment by monalishah** [comment]
MAR-2377 Jira created for it. Please do not update this GITHUB ticket anymore.
